rules/linux/process_creation/proc_creation_lnx_local_account.yml

```diff
@@ -28,10 +28,13 @@ detection:
             - '/tail'
             - '/vi'
             - '/vim'
+            - '/less'
+            - '/cp'
         CommandLine|contains:
             - '/etc/passwd'
             - '/etc/shadow'
             - '/etc/sudoers'
+            - '/etc/spwd.db'
```

Comment:
I didn't know `spwd.db` , It seems that there are also `/etc/pwd.db` 
https://man.cx/pwd_mkdb(8)

Reply:
Yes, during the attack simulation, I observed the spwd.db file. For this reason, I only mentioned it, but we can also add the other two files, namely /etc/pwd.db and /etc/master.passwd, to this rule.